### PR TITLE
chore: move globalLibraries configuration for the groovy init scripts to JCasC

### PR DESCRIPTION
**Pull request by lemeurherve**
Followup of https://github.com/jenkins-infra/jenkins-infra/pull/2047

### CODE REVIEW

**Comment by lemeurherve** [comment]
Here is the config I've got with these modifications on a local Jenkins in Vagrant:
image
WDYT @dduportal @timja LGTY?

Note that I've omitted the cache setting for now.

**Review by dduportal — APPROVED** [review]
That look really good ! Thanks a lot !

File: dist/profile/templates/buildmaster/casc/global-libraries.yaml.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+unclassified:
+  globalLibraries:
+    libraries:
+      - defaultVersion: "master"
+        implicit: true
+        includeInChangesets: false
+        name: "pipeline-library"
```
Comment (timja) [inline dist/profile/templates/buildmaster/casc/global-libraries.yaml.erb:7]: you're missing the caching config from the other file?
```suggestion
        name: "pipeline-library"
        cachingConfiguration:
          refreshTimeMinutes: 180
```

Comment (lemeurherve) [reply dist/profile/templates/buildmaster/casc/global-libraries.yaml.erb:7]: > [...] I've omitted the cache setting for now.

I'll do it in another PR.

Comment (dduportal) [reply dist/profile/templates/buildmaster/casc/global-libraries.yaml.erb:7]: I'm resolving conversation to allow merging

Comment (dduportal) [reply dist/profile/templates/buildmaster/casc/global-libraries.yaml.erb:7]: https://github.com/jenkins-infra/helpdesk/issues/2751 for tracking cache

**Pull request merged**